### PR TITLE
Test/mocking stubbing input test

### PR DESCRIPTION
**Pull request by RosaTheDev**
… data

x Wrote Tests ____ Implemented ____ Reviewed

## Necessary checkmarks:

    [x] All Tests are Passing in all environments

    [] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description:  Wrote tests for the text input and the mocking of data 
    
## Requested feedback: 
    
    I'd like feedback on... 

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [] No Tests have been changed
    [x] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [] My code has no binding.pry's
    [x] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: cypress/fixtures/allCryptids.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (GraceGardner) [inline cypress/fixtures/allCryptids.json:1]: These fixtures are awesome

File: cypress/integration/allsightings-test.js [hunk 2]
```diff
@@ -1,9 +1,21 @@
+import '../fixtures/allSightings.json'
 describe('All sightings page cypress test', () => {
 
   beforeEach(() => {
+    cy.intercept({
```
Comment (GraceGardner) [inline cypress/integration/allsightings-test.js:5]: Thanks for figuring out these intercepts

File: cypress/integration/filter-and-search-sightings-test.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    cy.get('.sightings-button').click()
+    cy.get('input[type="text"]').type('C')
+    cy.get('input[type="text"]').type('o')
+    cy.get('input[type="text"]').type('l')
+    cy.get('input[type="text"]').type('o')
+    cy.get('input[type="text"]').type('r')
+    cy.get('input[type="text"]').type('a')
+    cy.get('input[type="text"]').type('d')
+    cy.get('input[type="text"]').type('o')
+  })
```
Comment (GraceGardner) [inline cypress/integration/filter-and-search-sightings-test.js:25]: I might add testing that checks for specific cards when the search is finished, just to confirm that it's rendering what we expect.

**Review by GraceGardner — APPROVED** [review]
Looks great! I think all that is still needed testing wise is to test for Error handling :) Nice work!

File: cypress/fixtures/singleCryptid.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "data": {
```
Comment (krdurrett) [inline cypress/fixtures/singleCryptid.json:2]: Thanks for taking care of these fixtures/intercepts!

**Review by krdurrett — APPROVED** [review]
Strong work Richard!

**Pull request merged**